### PR TITLE
Allows to pass -k to disable assets compilation

### PR DESCRIPTION
**Pull request by paganotoni**
This is related to #700

### CODE REVIEW

File: buffalo/cmd/build/builder.go [hunk 1]
```diff
@@ -38,6 +38,12 @@ func New(ctx context.Context, opts Options) *Builder {
 		b.buildBin,
 	}
 
+	if b.Options.SkipAssets {
```
Comment (markbates) [inline buffalo/cmd/build/builder.go]: This won't work correctly. If you do this here then the templates, and anything else that uses `packr` won't run. The correct thing to do is to put the logic in the `buildAssets` method.

Comment (paganotoni) [reply buffalo/cmd/build/builder.go]: I see, ok, i changed the code to do the check inside the buildAssets function, LMK if it makes sense.

**Comment by markbates** [comment]
I think the logic looks good. My advice would be to keep the flag as is, but change the name of the option to be `WithAssets` that way the logic reads easier. Does that make sense?

**Comment by paganotoni** [comment]
Sounds good, going for it, thanks for all the patience and guidance on this one Mark

**Pull request merged**